src/array.rs

```diff
@@ -72,18 +72,19 @@ impl<'a, A: Clone> TreeLike for BTreeSlice<'a, A> {
 
 /// Partition of a slice into blocks of (lengths of) powers of two.
 ///
-/// The blocks start at (length) `N` and decrease to one in order.
-/// Depending on the (length of the) slice, some blocks might be empty.
+/// ## Partition of less than 2^(n + 1) elements
```

Comment:
In 7f178c43b0595e09eb099228224d41556342ea0f:

Should not have the `##` and the first line should end with a `.`.

Reply:
Oh never mind, I am misreading the diff.